### PR TITLE
[hotfix][docs] Fix the wrong characters in the document(building.md,building.zh.md)

### PR DESCRIPTION
**Pull request by yangjf2019**
## What is the purpose of the change

* Fix the wrong characters in the document(building.md,building.zh.md)

## Brief change log

*I modified the "docs/flinkDev/building.md" and  "docs/flinkDev/building.zh.md"  file .*

## Verifying this change

*I have verified the changes by starting the build script in preview mode.*

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)
  - If yes, how is the feature documented? (not applicable)

Hi, @wuchong @zentol Please take a look .Thank you very much.

### CODE REVIEW

File: docs/flinkDev/building.md [hunk 1]
```diff
@@ -128,7 +128,7 @@ mvn clean install -Dhadoop.version=2.6.5-custom
 After this step is complete, follow the steps for [Pre-bundled versions](#pre-bundled-versions).
 
 To build Flink against a vendor specific Hadoop version, additionally activate -Pvendor-repos` profile when building
```
Comment (wuchong) [inline docs/flinkDev/building.md]:
```suggestion
To build Flink against a vendor specific Hadoop version, additionally activate `-Pvendor-repos` profile when building
```

File: docs/flinkDev/building.md [hunk 2]
```diff
@@ -128,7 +128,7 @@ mvn clean install -Dhadoop.version=2.6.5-custom
...
-`flink-shaded`.
+`flink-shaded:
```
Comment (wuchong) [inline docs/flinkDev/building.md]:
```suggestion
flink-shaded:
```

File: docs/flinkDev/building.md (hunk 1 above)
Comment (yangjf2019) [reply docs/flinkDev/building.md]: Thanks ,I will update it .

File: docs/flinkDev/building.md (hunk 2 above)
Comment (yangjf2019) [reply docs/flinkDev/building.md]: Thanks ,I will update it .

File: docs/flinkDev/building.md [hunk 3]
```diff
@@ -127,8 +127,7 @@ mvn clean install -Dhadoop.version=2.6.5-custom
 
 After this step is complete, follow the steps for [Pre-bundled versions](#pre-bundled-versions).
 
-To build Flink against a vendor specific Hadoop version, additionally activate -Pvendor-repos` profile when building
-`flink-shaded`.
+To build Flink against a vendor specific Hadoop version, additionally `activate -Pvendor-repos` profile when building flink-shaded:
```
Comment (StephanEwen) [inline docs/flinkDev/building.md]: The tick here seems to be in the wrong position, still.

Comment (yangjf2019) [reply docs/flinkDev/building.md]: Oh, my god! Sorry, this is my fault.

**Pull request merged**